src/app/api/composio/connect/disconnect/route.ts

```
@@ -0,0 +1,102 @@
+          { status: 403 },
+        );
+      }
+    } catch (fetchError: unknown) {
+      // Handle cases where the connection might not be found by `get` (e.g., already deleted, invalid ID)
+      console.error(`Error fetching connection ${connectedAccountId} for verification:`, fetchError);
+
+      let errorMessage = "Failed to verify connection ownership.";
+      let errorStatus = 500;
+
+      if (typeof fetchError === 'object' && fetchError !== null) {
+        const status = (fetchError as any).status; // Potentially unsafe, but common for SDK errors
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)